modules/tspec/fmt.go

```diff
@@ -0,0 +1,76 @@
+package tspec
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"unicode/utf8"
+
+	"github.com/gruntwork-io/terratest/modules/tspec/colors"
+	"github.com/gruntwork-io/terratest/modules/tspec/formatters"
+	internal_fmt "github.com/gruntwork-io/terratest/modules/tspec/internal/formatters"
+	"github.com/gruntwork-io/terratest/modules/tspec/internal/models"
+	"github.com/gruntwork-io/terratest/modules/tspec/internal/storage"
+)
+
+// FindFmt searches available formatters registered
+// and returns FormaterFunc matched by given
+// format name or nil otherwise
+func FindFmt(name string) FormatterFunc {
+	return formatters.FindFmt(name)
+}
+
+// Format registers a feature suite output
+// formatter by given name, description and
+// FormatterFunc constructor function, to initialize
+// formatter with the output recorder.
+func Format(name, description string, f FormatterFunc) {
+	formatters.Format(name, description, f)
+}
+
+// AvailableFormatters gives a map of all
+// formatters registered with their name as key
+// and description as value
+func AvailableFormatters() map[string]string {
+	return formatters.AvailableFormatters()
+}
+
+// Formatter is an interface for feature runner
+// output summary presentation.
+//
+// New formatters may be created to represent
+// suite results in different ways. These new
+// formatters needs to be registered with a
+// tspec.Format function call
+type Formatter = formatters.Formatter
+
+type storageFormatter interface {
+	SetStorage(*storage.Storage)
+}
+
+// FormatterFunc builds a formatter with given
+// suite name and io.Writer to record output
+type FormatterFunc = formatters.FormatterFunc
+
+func printStepDefinitions(steps []*models.StepDefinition, w io.Writer) {
```

Comment:
What is this formatter stuff? And step definitions? I think I'm missing some context on how this stuff all comes together.